ZeeHomeScreen/Classes/Components/Banner/BannerCellViewController.swift

```diff
@@ -28,7 +26,7 @@ class BannerCellViewController : UIViewController, ComponentProtocol, ComponentD
         
         @IBOutlet weak var contentView: UIView!
         
-        private var adPresenter: ZPAdPresenterProtocol?
+        var adPresenter: ZPAdPresenterProtocol?
```

Comment:
Let's keep this encapsulated. Each Banner view controller can receive a notification and pause or resume the video ads.